### ISSUE TITLE
Suggestion: make a "2P BATTLE" button inactive in the main menu if both 2P Keyboard and Gamepad disabled

### ISSUE THREAD

**Issue by RostovShev03** [shaddatic/sa2b-input-controls#1]
When both 2P Keyboard and 2P Gamepad controls disabled, a "2P BATTLE" button still active but nothing happens. I'd like to see this button inactive when they disabled (like in console versions - Dreamcast, Gamecube and later HD re-releases for PS3 and Xbox 360)

**Comment by shaddatic** [comment]
Yeah, that's on the table to be done. I didn't want to do it on the first update as I wanted it to be as stable as possible, and messing with menus could cause who knows what issues
